test/Spec.hs

```diff
@@ -6,6 +6,10 @@ import Hedgehog (checkParallel)
 import System.IO (hSetEncoding, utf8)
 import Test.Relude.Property (hedgehogTestList)
 
+import qualified Control.Applicative as App
+
+foo = App.empty
```

Comment:
🤔

Reply:
Sorry for that, leftover from testing...